po/pt_BR.po

```diff
@@ -298,23 +308,41 @@ msgid ""
 "You can add and connect to an IRC server with /server and /connect commands "
 "(see /help server)."
 msgstr ""
+"Bem-vindo ao WeeChat!\n"
+"\n"
+"Se você está iniciando no WeeChat é recomendável ler pelo menos o"
+"guia de início rápido e o guia de usuário, se você tiver algum tempo. Neles você"
+"entenderá os principais conceitos do WeeChat.\n"
+"Todos os documentos do WeeChat estão disponíveis em: https://weechat.org/doc\n"
+"\n"
+"Além disso, há um comando de ajuda no chat, /help, disponível em todos os comandos e opções (use"
+"Tecla Tab para completar o nome).\n"
+"O comando /fset pode te ajudar a personalizar o WeeChat.\n"
+"\n"
+"Você pode adicionar e conectar-se a um servidor IRC com os comandos /server e /connect"
+"(consulte /help na aba servidor para ajuda)."
 
 #. TRANSLATORS: the "under %s" can be "under screen" or "under tmux"
 #, c-format
 msgid ""
 "%sWarning: WeeChat is running under %s and $TERM is \"%s\", which can cause "
 "display bugs; $TERM should be set to one of these values: %s"
 msgstr ""
+"%sAviso: WeeChat está rodando em plano de fundo %s e $TERM está como \"%s\", o que pode causar "
+"erros no display; $TERM deveria ser alterado para um desses valores: %s"
 
 #, c-format
 msgid "%sYou should add this line in the file %s:  %s"
 msgstr ""
+"%sVocê deveria adicionar esta linha em seu arquivo %s:  &s"
```

Comment:
The string ends with `%s`, not `&s`, this makes tests fail (formats of strings in translations are checked).